### DISCUSSION TITLE
x86 is broken, who uses it ?

### DISCUSSION THREAD

**Discussion by hannesa2** [gitx/gitx#419] [Polls]
As x86 is broken, and I'm not sure if the afford is a good investment. So it's interesting who uses it and is interested on further support.
If you vote for Yes , please don't forget to contribute to help to solve it.

**Comment by claytonrcarter** [comment]
I still use GitX on x86, and would love for it to stay up to date. I understand the concern though; with limited support time and difficult builds, it probably doesn’t make sense to put too much effort into x86 support. The need will continue to drop with time too, as those of us with x86 laptops eventually upgrade.

**Comment by insha** [comment]
Do we have numbers on which macOS versions GitX is being? We may want consider raising the minimum macOS, if that would help in getting people to contribute.

**Reply by hannesa2** [reply]
I don't know such a info.
This is all known to me https://formulae.brew.sh/cask/gitx#default
<img width="660" alt="image" src="https://github.com/gitx/gitx/assets/3314607/fb3eea69-e148-404f-b2e6-ec8260053ea5">

Keep in mind, that a lot of installations are not done with brew

**Reply by insha** [reply]
Cool. Thank you. I'll start a poll and see if we can get some more context into these numbers.

**Comment by hannesa2** [comment]
Because since one year CI for x86 is broken, and nobody was able to fix it, I drop x86 support https://github.com/gitx/gitx/pull/431
If you want to have further x86 support, please remove comment and try to fix it.
